lib/steep/cli.rb

```diff
@@ -20,6 +20,10 @@ def self.available_commands
     end
 
     def setup_global_options
+      OptionParser.new do |opts|
+        opts.version = Steep::VERSION
+      end.parse(argv)
+
```

Comment:
I think this is a minimal implementation, but it is confusing me the behavior which `OptionParser#parse` exits the program when it find a `--verison` option. 🤔

Reply:
I think defining `::Version` is better to handle other options in sub commands.

Reply:
I defined `::Version`. Is this code correct?

Reply:
I mean you don't have to do anything in `setup_global_options`. Just leave it empty.

Reply:
When I try to leave `setup_global_options` empty, then `--version` or `-v` options don't work. 😢 

```
$ ./exe/steep --version
Unknown command: --version
  available commands: check, validate, annotations, scaffold, interface, version
```

Here is my change:

```diff
     def setup_global_options
-      version = OptionParser.new.version
-      if version
-        stdout.puts version
-        return false
-      end
-
       true
     end
 ```

Reply:
Ah, got it.